receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -126,7 +128,13 @@ func (s *splunkScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	wg.Wait()
 	close(errChan)
 	errs = <-errOut
-	return s.mb.Emit(), errs.Combine()
+	combinedErrs := errs.Combine()
+	if combinedErrs != nil {
+		s.settings.Logger.Error(fmt.Sprintf("Scrape errors: %v", combinedErrs.Error()))
+		s.failedScrapes++
```

Comment:
Considering that combined can be great than one, would it more advisable to increment failed scrapes by the amount of errors seen?

Reply:
I wanted to do this, But the errors are stored as a [ScrapeError](https://github.com/open-telemetry/opentelemetry-collector/blob/f9197adbcc59454cd665252806ba4a68c97efe47/scraper/scrapererror/scrapeerror.go#L12), which doesn't expose its errs array for me to get a length on.

Reply:
I tried working around this by casting the result of [Combine()](https://github.com/open-telemetry/opentelemetry-collector/blob/f9197adbcc59454cd665252806ba4a68c97efe47/scraper/scrapererror/scrapeerror.go#L29) to [PartialScrapeError](https://github.com/open-telemetry/opentelemetry-collector/blob/main/scraper/scrapererror/partialscrapeerror.go#L10) in order to retrieve the value of `Failed`, but I was running into errors and it didn't seem worth the trouble to me.